client-misk/src/main/kotlin/app/cash/backfila/client/misk/BackfilaClientModule.kt

```diff
@@ -35,22 +37,18 @@ annotation class ForBackfila
 
 class BackfilaClientModule(
   private val config: BackfilaClientConfig,
-  private val backfills: List<Class<out Backfill>>
+  private val backfills: List<KClass<out Backfill>>
```

Comment:
what's the reason to convert to KClass? Curious

Reply:
Query uses KClass further down. I figured in Kotlin we might want to use KClass over Class?

Reply:
For a Kotlin client I'd also lean with `KClass`

Reply:
They are pretty much interchangeable. Jesse didn't seem to care but agreed with my reasoning.